### PR TITLE
[plugin.video.curiositystream] 2.0.0

### PR DESCRIPTION
**Pull request by GMaxera**
Version 2.0.0 of Curiositystream plugin.

Changes:
- Added search
- Added subtitles
- Removed "News this week"
- Added Add/Remove watchlist context menu action
- Improvements on video description

### CODE REVIEW

File: plugin.video.curiositystream/resources/lib/router.py [hunk 1]
```diff
@@ -66,13 +67,22 @@ def route(self, query):
...
             self._list_media_or_collections(action, parent, page)
         elif action == "play":
             self._play_media(params["media"])
+        elif action == "change_watchlist":
+            self._change_watchlist(
+                params["media"], params["is_bookmarked"], params["is_collection"]
+            )
+        elif action == "search":
+            query = params.get("query", None)
+            query = xbmcgui.Dialog().input("Search") if not query else query
```
Comment (enen92) [inline plugin.video.curiositystream/resources/lib/router.py:82]: Please localize "Search"

File: plugin.video.curiositystream/resources/lib/router.py [hunk 2]
```diff
@@ -187,6 +197,21 @@ def pagination_entry(new_page, label):
             list_item.setArt(m["art"])
             list_item.setInfo("video", m["video"])
             list_item.setProperty("IsPlayable", "false" if m["is_folder"] else "true")
+            list_item.addContextMenuItems(
+                [
+                    (
+                        tr(30023) if m["is_bookmarked"] else tr(30024),
+                        "XBMC.RunPlugin({}?action=change_watchlist"
```
Comment (enen92) [inline plugin.video.curiositystream/resources/lib/router.py:204]: Drop xbmc. prefix

File: plugin.video.curiositystream/resources/lib/router.py [hunk 3]
```diff
@@ -202,5 +227,17 @@ def pagination_entry(new_page, label):
...
+
+    def _change_watchlist(self, media_id, is_bookmarked, is_collection):
+        self._cs_api.update_user_media(
+            media_id, is_bookmarked != "True", is_collection == "True"
+        )
+        dialog = xbmcgui.Dialog()
+        dialog.notification(
+            tr(30025), tr(30026) if is_bookmarked == "True" else tr(30027),
+        )
+        xbmc.executebuiltin("XBMC.Container.Refresh()")
```
Comment (enen92) [inline plugin.video.curiositystream/resources/lib/router.py:243]: drop xbmc. prefix

**Comment by GMaxera** [comment]
@enen92 Thank you for your review 👍 I've fixed the issues as you requested.
;-) No worries for any delay 👍

**Pull request merged**